ovs/matchflow.go

```diff
@@ -105,7 +105,7 @@ func (f *MatchFlow) MarshalText() ([]byte, error) {
 		b = append(b, ',')
 	}
 
-	if f.Cookie > 0 {
+	if f.Cookie > 0 || f.CookieMask > 0 {
```

Comment:
What was the issue that you ran into that led to this change? Could you give a little more context? TIA!

Reply:
Cookie and CookieMask are uint64. So currently there is on way to match zero cookie or a zero cookie with a non-zero mask. So this check will enable use to check for zero cookie match. For example: cookie=0x0000000000000000/0xffffffffffffffff